lib/fluent/plugin/kubernetes_metadata_common.rb

```diff
@@ -82,13 +82,28 @@ def parse_pod_metadata(pod_object)
         log.warn("parsing container meta information failed for: #{pod_object[:metadata][:namespace]}/#{pod_object[:metadata][:name]}: #{e}")
       end
 
+      controller_meta = {}
+      begin
+        key = 0
+        pod_object[:metadata][:ownerReferences]. each do |owner_reference|
+          controller_meta[key.to_s] = {
+            'kind' => owner_reference[:kind],
+            'name' => owner_reference[:name]
+          }
+          key += 1
+        end
+      rescue StandardError=>e
+        log.warn("parsing controller meta information failed for: #{pod_object[:metadata][:namespace]}/#{pod_object[:metadata][:name]}: #{e}")
+      end if @include_controller_metadata
+
       kubernetes_metadata = {
         'namespace_name' => pod_object[:metadata][:namespace],
         'pod_id' => pod_object[:metadata][:uid],
         'pod_name' => pod_object[:metadata][:name],
         'pod_ip' => pod_object[:status][:podIP],
         'containers' => syms_to_strs(container_meta),
-        'host' => pod_object[:spec][:nodeName]
+        'host' => pod_object[:spec][:nodeName],
+        'controller' => syms_to_strs(controller_meta)
```

Comment:
might this better be called "ownerrefs" or something of the like?  It otherwise looks good to me and could use at least one test

Reply:
Thanks for the quick review!

Do you want me to change the option name as well? from `include_controller_metadata` to `include_owner_metadata` or some such?

Reply:
Please update the config to make it clear what it is enabling

Reply:
@jcantrill alright, I think I have implemented all of your recommendations.

* I changed everything to be consistently `ownerrefs` rather than `controller`.
* I ensured that the existing tests were passing
* I added an additional test for this new attribute

Let me know if you want any further changes.